sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Azure.AI.MetricsAdvisor.csproj

```diff
@@ -3,6 +3,10 @@
     <AssemblyTitle>Microsoft Azure.AI.MetricsAdvisor client library</AssemblyTitle>
     <Version>1.0.0-beta.2</Version>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>
+      $(NoWarn);
+      AZC0007; <!-- https://github.com/Azure/azure-sdk-for-net/issues/16435 -->
+    </NoWarn>
```

Comment:
Blocked by https://github.com/Azure/azure-sdk-tools/issues/127#issuecomment-718993646.

I tried suppressing the warning locally, but it still fails when exporting the API.